packages/contact-center/task/tests/IncomingTask/incoming-task.presentational.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {render, screen, fireEvent, cleanup} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import IncomingTaskPresentational from '../../src/IncomingTask/incoming-task.presentational';
+
+describe('IncomingTaskPresentational', () => {
+  afterEach(cleanup);
+
+  it('renders "no task yet" when there is no current task', () => {
+    const props = {
+      currentTask: null,
+      accept: jest.fn(),
+      decline: jest.fn(),
+      isBrowser: true,
+      answered: false,
+      ended: false,
+      missed: false,
+    };
+
+    render(<IncomingTaskPresentational {...props} />);
+
+    const message = screen.getByText('no task yet');
+    expect(message).toBeInTheDocument();
+  });
+
+  it('renders incoming call details and buttons for browser option', () => {
```

Comment:
Change the description of the test as we are just checking the rendering

Reply:
addressed